chromadb/utils/embedding_functions.py

```diff
@@ -244,9 +248,20 @@ class ONNXMiniLM_L6_V2(EmbeddingFunction):
 
     # https://github.com/python/mypy/issues/7291 mypy makes you type the constructor if
     # no args
-    def __init__(self) -> None:
+    def __init__(self, preferred_providers: Optional[List[str]] = None) -> None:
```

Comment:
Is this a bad git history/merge?

Reply:
I think the rebase did this. Let me check.

Reply:
@HammadB, I did some cherry-picking and think I've got it all. Running local tests, and then I will close this one and raise a separate PR with a link to this one.